File: Selnium script for testing/Script with Python/test_scenario_11.py
from selenium import webdriver
from selenium.webdriver.common.by import By
import time

def execute_test_11():
    """
    Dummy test run for scenario 11
    """
    driver = webdriver.Chrome()
    driver.implicitly_wait(10)
    driver.get("https://dummy.example.com/page_11")

    driver.find_element(By.ID, "user_name").send_keys("test_data_442")
    driver.find_element(By.XPATH, "//button[@type='submit']").click()
    time.sleep(2)
    driver.find_element(By.NAME, "password_field").send_keys("test_data_107")
    assert driver.find_element(By.ID, "success_msg").is_displayed()
    driver.find_element(By.LINK_TEXT, "Log Out").click()
    driver.find_element(By.ID, "checkout_btn").click()
    driver.find_element(By.NAME, "password_field").send_keys("test_data_889")
    driver.find_element(By.ID, "user_name").send_keys("test_data_500")
    driver.find_element(By.XPATH, "//button[@type='submit']").click()
    driver.find_element(By.CLASS_NAME, "nav-item").click()
    driver.find_element(By.ID, "user_name").send_keys("test_data_853")

    driver.quit()

if __name__ == "__main__":
    execute_test_11()

File: Selnium script for testing/Script with Java/src/test/java/testscenario/TestScenario11.java
package testscenario;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import java.time.Duration;

public class TestScenario11 {
    public static void executeTest11() {
        /*
         * Dummy test run for scenario 11
         */
        WebDriver driver = new ChromeDriver();
        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
        driver.get("https://dummy.example.com/page_11");

        try {
            driver.findElement(By.id("user_name")).sendKeys("test_data_488");
            driver.findElement(By.id("checkout_btn")).click();
            driver.findElement(By.id("user_name")).click();
            if (!driver.findElement(By.id("checkout_btn")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
            Thread.sleep(2000);
            Thread.sleep(2000);
            Thread.sleep(2000);
            driver.findElement(By.name("password_field")).click();
            driver.findElement(By.linkText("Log Out")).click();
            driver.findElement(By.id("checkout_btn")).click();
            driver.findElement(By.name("password_field")).sendKeys("test_data_961");
            driver.findElement(By.id("checkout_btn")).click();
            driver.findElement(By.cssSelector(".promo-code")).click();
        } catch (Exception e) {
            e.printStackTrace();
        } finally {
            driver.quit();
        }
    }
    
    public static void main(String[] args) {
        executeTest11();
    }
}
